extensions/ql-vscode/src/model-editor/model-editor-module.ts

```diff
@@ -173,4 +174,24 @@ export class ModelEditorModule extends DisposableObject {
   private async initialize(): Promise<void> {
     await ensureDir(this.queryStorageDir);
   }
+
+  private async showMethod(usage: Usage): Promise<void> {
+    await this.methodsUsagePanel.revealItem(usage);
+
+    // For now, just construct a dummy method and show it in the method modeling panel
```

Comment:
> I've not wired up the actual method information yet because it requires some refactoring to make that available (we only have the `Usage` at the minute). This will be done in a following PR.

I agree with this proposed approach. We can add the `ExternalApiUsage` to the `jumpToUsage` message from the model editor view, so then we have both the method and the usage available. I was planning / expecting we'd do that anyway, even before you started on this new modelling panel.